api/v1/policy_types.go

```diff
@@ -125,6 +138,13 @@ type PolicyStatus struct {
 	Details         []*DetailsPerTemplate `json:"details,omitempty"`   // used by replicated policy
 }
 
+// ReplicatedPolicyStatus defines the replicated policy status
+type ReplicatedPolicyStatus struct {
```

Comment:
Could you please move this to `api/v1beta1/policyautomation_types.go` since replicated policies actually use `PolicyStatus`?